igor-web/src/main/java/com/netflix/spinnaker/igor/gcb/GoogleCloudBuildParser.java

```diff
@@ -49,14 +49,12 @@
   }
 
   String serialize(Object input) {
-    try {
-      Writer writer = new StringWriter();
-      JsonGenerator generator = jacksonFactory.createJsonGenerator(writer);
+    Writer writer = new StringWriter();
+    try (JsonGenerator generator = jacksonFactory.createJsonGenerator(writer)) {
       generator.serialize(input);
-      generator.flush();
-      return writer.toString();
     } catch (IOException e) {
       throw new RuntimeException(e);
```

Comment:
idea for future cleanup: `throw new UncheckedIOException(e)` (a new class in Java 8)

But I see this pattern is used all over this package, and it's of questionable value, so whatever

Reply:
Nice, I didn't realize that existed!  I replaced all of the `RuntimeException`s that were wrapping `IOException`s with `UncheckedIOException`s.